packages/components/src/icons/docs/Icon.stories.mdx

```diff
@@ -19,9 +19,13 @@ import IndexFileUsage from "./IndexFileUsage.mdx";
     githubPath="/packages/components/src/icons/src"
 />
 
-## Accessibility
+## Guidelines
 
-When applicable, an accessible name should be provided through `aria-label` to indicate the **purpose** of the icon in it's **current context**. For more information see [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute).
+### Accessibility
+
+- When applicable, an accessible name should be provided through `aria-label` to indicate the **purpose** of the icon in it's **current context**.
```

Comment:
I would also add that if the icon is not adding any information and doesn't need to be announced by a screen reader, just like Img uses an empty alt, icons should use `aria-hidden="true"` to be ignored by screenreaders while still displaying in the UI.
cf: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden#example